app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="container">
+  <div class="child mt-2">
+    <div class="text-center mb-3">
+      <h2>Sign up</h2>
+    </div>
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+
+      <div class="field mb-3">
+        <br />
+        <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder:'Name Here!', class:'text-input form-control form-control-lg' %>
+      </div>
+
+      <div class="field mb-3">
+        <br />
+        <%= f.email_field :email, autocomplete: "email", placeholder:'Email Here!', class:'text-input form-control form-control-lg' %>
+      </div>
+
+      <div class="field mb-3">
+        <% if @minimum_password_length %>
+        <% end %><br />
+        <%= f.password_field :password, autocomplete: "new-password", placeholder:'Password Here!', class:'text-input form-control form-control-lg' %>
+      </div>
+
+      <div class="field mb-3">
+        <br />
+        <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder:'Confirmation Password Here!', class:'text-input form-control form-control-lg' %>
+      </div>
+
+      <div class="actions mb-3">
+        <%= f.submit "Sign up", class: 'btn btn-outline-primary' %>
+      </div>
+    <% end %>
+
+    <%= render "devise/shared/links" %>
+  </div>
+</div>
```

Comment:
- [ ] Please implement the changes suggested by the previous reviewer for this _**Sign Up**_ page. From the design template, the page should have a navigation bar, with the `background-color` of `#3778c2` and an icon to go back to the previous page. Also, notice the structure of the form inputs used and the colours that were used on the page. It would be nice for you to match your design to that of the design template provided for this project. 👍 😎 

 | Your Design | Design Template |
| --------------| -------------------|
| ![image](https://user-images.githubusercontent.com/77106962/236007377-5dc3603d-7825-4f64-8891-003f3ca8c02e.png) | ![image](https://user-images.githubusercontent.com/77106962/236007469-b8ea948e-aa46-4bee-8659-89450c16bfdf.png) |